pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -440,7 +440,40 @@ class ClassTypeHintMatch:
                 ),
             ],
         ),
-    ]
+    ],
+}
+_PROPERTY_MATCH: dict[str, list[ClassTypeHintMatch]] = {
```

Comment:
Can you add a comment explaining that these are normally checked by mypy and only with `--ignore-missing-annotations false` are also checked with pylint.